test/test-cases/react-css-modules/__snapshots__/react-css-modules.test.js.snap

```diff
@@ -19,6 +19,7 @@ Object {
 ",
   "main.js": "CONTENTS IGNORED IN SNAPSHOT TEST",
   "render.js": "CONTENTS IGNORED IN SNAPSHOT TEST",
-  "style.css": ".lr46_gr{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex}",
+  "style.css": ".lr46_gr{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex}
+",
```

Comment:
This seems like it may be an issue? Don't think there should be a newline here.

Reply:
I wonder if its the behaviour of `mini-css-extract-plugin`. I will have a look later

Reply:
Hi @mattsjones. Sorry for taking a very long time. I tested the build on SRP. The CSS file generated seems fine. I am not sure how the new line is added. After running the test script multiple times, I am pretty sure its added by the library